cmd/bundle/main_test.go

```diff
@@ -32,6 +35,40 @@ var _ = Describe("Bundle Loader", func() {
 		f(path)
 	}
 
+	withTempFile := func(pattern string, f func(filename string)) {
```

Comment:
Nit: Both styles are ok, I was just wondering since the previous two helper functions used the `var` style (mostly to be a bit more visible), I would like to have the four new helper functions defined in the same way.
```suggestion
	var withTempFile = func(pattern string, f func(filename string)) {
```